.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,95 @@
+name: Build wheels
```

Comment:
```suggestion
name: Build Windows wheels
```

suggestion: also change filename to `build_windows_wheels.yml`; I think even if we can use much of this same approach for MacOS / Linux, that we will probably want to keep them in separate files & top-level builds.